examples/creatives/banner_nonlinear.html

```diff
@@ -0,0 +1,57 @@
+<html>
+  <head>
+    <script src="../simid_protocol.js"> </script>
+    <script src="base_simid_creative.js"> </script>
+    <script src="banner_nonlinear.js"> </script>
+  </head>
+  <style>
+    #close_ad {
+      color: red;
+      border: solid white 1px;
+      font-size: 14;
+      float: right;
+    }
+    #minimize_ad {
+      color:rgb(78, 78, 78);
+      border: solid white 1px;
+      font-size: 14;
+      float: right;
+    }
+    button {
+      cursor: pointer;
+    }
+
+    body {
```

Comment:
be consistent in the spacing - either have a line between each selector or none

Reply:
Good call, looks like the other creatives go without the extra space so changed to follow that standard.